### ISSUE TITLE
command 'vandelay.selectImport' not found

### ISSUE THREAD

**Issue by greentag** [ericbiewener/vscode-vandelay#22]
First of all thank you for this incredible extension.

I was using vandelay and it was working fine. Week ago I started getting "command 'vandelay.selectImport' not found" message when I run "Import..." command.
I already have "vandelay-py.js" file in root folder.

Thank you in advance.

**Comment by ericbiewener** [comment]
A few troubleshooting questions/steps:

1. What version of Vandelay do you have?
2. Do you see any error alerts in the bottom right of VS Code when first opening your project?
3. Run the `Toggle Developer Tools` command from the command palette. Do you see any errors? And do you see any of the following lines?

```
[Extension Host] Loading vandelay config file from <project_root>/vandelay-js.js
[Extension Host] Loading vandelay config file from <project_root>/vandelay-py.js
[Extension Host] Vandelay language registered: py
```

I'm guessing you won't see the last line, since it seems like Python language support is not being successfully initialized by Vandelay.

4. Are you able to provide a link to your repo? If not, please post the contents of your `vandelay-py.js` file
5. If you start a new Python project with a simple vandelay-py.js file, does that work?

**Comment by greentag** [comment]
Thank you for instant response :)
1. Version is 4.0.0
2. Yes I am getting message "Cannot read property 'push' of undefined" (see details below)
3. As you you said I can see only first two lines.
4. Sorry I can't. See bellow.
5. No

<img width="1634" alt="Screen Shot 2019-08-10 at 23 14 45" src="https://user-images.githubusercontent.com/11061119/62825423-b8355d80-bbc4-11e9-8f5c-2369784d1969.png">

**Comment by ericbiewener** [comment]
Can you create a repo with a basic test case that is failing since you said you're not even able to start a new Python project with a simple vandelay-py.js file? Post the link to the repo here so that I can pull it down and poke around.

**Comment by greentag** [comment]
Sure: https://github.com/greentag/vandelay/

**Comment by ericbiewener** [comment]
Ok, the bug is now fixed in v4.0.1 and available from the VS Code marketplace.
